ember-app/app/components/hb-milestone.js

```diff
@@ -9,32 +9,42 @@ var HbMilestoneComponent = Ember.Component.extend({
   isOpen: function(){
     return false;
   }.property(),
-
   listItems: function () {
-
     return this.get("milestones")
     .filter(function(item) {
       var term = this.get("filterMilestones") || "";
       return item.data.title.toLowerCase().indexOf(term.toLowerCase() || item.data.title.toLowerCase()) !== -1;
     }.bind(this))
-    .map(function(item) {
+    .sort(function(a, b) {
+
+      // always put selected on top
+      if(a.data.title === this.get('selected.title')) {
+        return -1;
+      }
+      if(b.data.title === this.get('selected.title')) {
+        return 1;
+      }
 
+      return a.data.title.localeCompare(b.data.title);
+    }.bind(this))
+    .map(function(item) {
       return this.ListItem.create({
-        selected: item.data.id === this.get("selected.id"),
+        selector: this,
```

Comment:
the list item is mapped off the repo milestones, but only one can be selected because it's a single property. 

item.selected was being mapped off of the selectors selected property, but I don't want that to immediately cause a recomputed of the list items, UX wise is weird to have the milestone selected jump to the top as soon as you click it...